lib/floe/workflow/state.rb

```diff
@@ -90,10 +90,6 @@ def ready?
         !started? || !running?
       end
 
-      def finished?
-        context.state.key?("FinishedTime")
-      end
-
```

Comment:
Hm even if we don't end up calling this internally I think it is a nice interface method for external callers.